config/helm/aws-node-termination-handler/templates/daemonset.windows.yaml

```diff
@@ -89,6 +89,8 @@ spec:
             - name: INSTANCE_METADATA_URL
               value: {{ . | quote }}
             {{- end }}
+            - name: CHECK_ASG_TAG_BEFORE_DRAINING
+              value: {{ .Values.checkASGTagBeforeDraining | quote }}
```

Comment:
Could you move this up below line 79? Keeping the parameters in the same order will make it easier to compare the linux and windows versions of this file in the future.

Reply:
sure, no big deal :)